frontend/src/scenes/plugins/tabs/installed/InstalledTab.tsx

```diff
@@ -158,7 +158,15 @@ export function InstalledTab(): JSX.Element {
                                 `Enabled plugins (${enabledPlugins.length})`
                             )
                         }
-                        buttons={rearranging ? rearrangingButtons : upgradeButton}
+                        buttons={
+                            rearranging ? (
+                                rearrangingButtons
+                            ) : pluginsNeedingUpdates.length === 0 ? (
+                                upgradeButton
+                            ) : (
+                                <div />
```

Comment:
`null` would be more better than an empty element, if possible

Reply:
That won't work due to some strict type. `<></>` did the trick though